src/tools/initialize.ts

```diff
@@ -52,10 +52,17 @@ export default async function initializeSandbox({
     `mcp-creation-timestamp=${creationTimestamp}`,
   ];
   const labelArgs = labels.map((label) => `--label "${label}"`).join(' ');
+  const memFlag = process.env.SANDBOX_MEMORY_LIMIT
+    ? `--memory ${process.env.SANDBOX_MEMORY_LIMIT}`
+    : '';
+
+  const cpuFlag = process.env.SANDBOX_CPU_LIMIT
+    ? `--cpus ${process.env.SANDBOX_CPU_LIMIT}`
+    : '';
 
   try {
     execSync(
-      `docker run -d ${portOption} --memory 512m --cpus 1 ` +
+      `docker run -d ${portOption} ${memFlag} ${cpuFlag} ` +
```

Comment:
Also, in case these values are not set explicitly by the user, can we have some meaningful defaults, based on the image? For example, the node slim might be something like 512m and 1cpu, while the node-chart optimized image and the playwright image might require 2cpu and 2gb by default. 

It would be nice to have a function that, given the choose image, returns some meaningful defaults - in case the user didn't do an override through the env vars